### PR TITLE
Assignment

### PR DESCRIPTION
**Pull request by parkerfoshay**
here is my homework lab

### CODE REVIEW

File: posts/directions.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+* Here is the [PDF version of the slides](./Slides/MongoDB-Slide-Deck.pdf) and the [slide deck presentation on Google](https://docs.google.com/presentation/d/194re6kxDcqaOAHMuKarTOMshiwjR-OOTtNbVbmLt2NM/edit?usp=sharing)
+
+* The five minute activity is in the [Activity](./Activity/explainer.md) folder.
+
+    * The activity has the learner use what they learned from the presentation in conjuction with the MongoDB documentation.
+
+    * This gives the learner the opportunity to inspect the documentation on their own which is more akin to what a software engineer will do daily.
+
+    * The learner also has the added benefit of putting what they just learned into their own words.
```
Comment (sevans924) [inline posts/directions.md:16]: Add another point below

**Review by sevans924 — APPROVED** [review]
LGTM!

File: posts/directions.md (hunk 1 above)
Comment (nielsenjared) [reply posts/directions.md:16]: Good point!

**Comment by nielsenjared** [comment]
LGTM!

File: posts/directions.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+title: 'Assignment'
+date: '2020-01-09'
+---
+
+# Parker Faucher's Take Home Assignment
+
+* Here is the [PDF version of the slides](./Slides/MongoDB-Slide-Deck.pdf) and the [slide deck presentation on Google](https://docs.google.com/presentation/d/194re6kxDcqaOAHMuKarTOMshiwjR-OOTtNbVbmLt2NM/edit?usp=sharing)
```
Comment (nielsenjared) [inline posts/directions.md:8]: Hey! This is the wrong URL, can you update?

**Pull request merged**